src/common/winstore.h

```diff
@@ -1,3 +1,15 @@
 #pragma once
 
-bool running_as_packaged();
+#include <future>
+
+#include <winrt/Windows.ApplicationModel.h>
+
+
+namespace winstore
+{
+    using winrt::Windows::ApplicationModel::StartupTaskState;
+
+    bool running_as_packaged();
+    std::future<void> switch_startup_task_state_async(const bool enabled);
+    std::future<StartupTaskState> get_startup_task_status_async();
+}
```

Comment:
missing new line

Reply:
Since we use .get() and .wait() anyway on the futures, do we need them?

Reply:
@bzoz I think we'll need them when we're going to move to WinUI, which follows a similar style as WinRT framework, i.e. "everything slower than 30ms must be asynchronous".